x-pack/legacy/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/categorization_detector/detector_cards.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { EuiFlexItem, EuiCard } from '@elastic/eui';
+
+interface CardProps {
+  onClick: () => void;
+  isSelected: boolean;
+}
+
+export const CountCard: FC<CardProps> = ({ onClick, isSelected }) => (
+  <EuiFlexItem>
+    <EuiCard
+      title={i18n.translate(
+        'xpack.ml.newJob.wizard.pickFieldsStep.categorizationDetectorSelect.countCard.title',
+        {
+          defaultMessage: 'Count',
+        }
+      )}
+      description={
+        <>
+          <FormattedMessage
+            id="xpack.ml.newJob.wizard.pickFieldsStep.categorizationDetectorSelect.countCard.description"
+            defaultMessage="Look for anomalies in the number of categories."
```

Comment:
This should be changed to `Look for anomalies in the event rate of a particular category.`